src/main/java/duke/main/Utils.java

```diff
@@ -0,0 +1,192 @@
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void welcomeMessage(){
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm duke.main.Duke\nWhat can I do for you?");
+    }
+
+    public static int validateCommand(String line){
+        if ((Arrays.asList(VALID_COMMANDS).contains(line))) {
+            return 1;
+        } else {
+            return 0;
+        }
+    }
+    public static int validateParameter(String line) {
+        int indexOfSlash = -1;
+        if (line.split(" ")[0].equals("deadline")) {
+            indexOfSlash = line.indexOf("/by");
+        } else if (line.split(" ")[0].equals("event")) {
+            indexOfSlash = line.indexOf("/at");
+        }
+        return indexOfSlash;
+    }
+    public static void createFile(){
+        try {
+            File myObj = new File("list.txt");
+            if (myObj.createNewFile()) {
+                System.out.println("File created: " + myObj.getName());
+            } else {
+                System.out.println("File already exists.");
+            }
+        } catch (IOException e) {
+            System.out.println("An error occurred.");
+            e.printStackTrace();
+        }
+    }
+    public static void writeToFile() throws IOException {
+        createFile();
+        FileWriter fw = new FileWriter("list.txt");
+        for (int i=0; i<Task.getNumOfTasks(); i++) {
+            ArrayList<Task> buffer = Task.getList();
+            switch (buffer.get(i).getType()) {
+
+            case ("T"):
+                fw.write(buffer.get(i).getType() + "\t" + buffer.get(i).isDone() + "\t" + buffer.get(i).getDescription() + "\n");
+                break;
+            case ("D"):
+                Deadline temp1 = (Deadline)(buffer.get(i));
+                fw.write(buffer.get(i).getType() + "\t" + buffer.get(i).isDone() + "\t" + buffer.get(i).getDescription() + "\t" + temp1.getBy() + "\n");
+                break;
+            case ("E"):
+                Event temp2 = (Event) (buffer.get(i));
+                fw.write(buffer.get(i).getType() + "\t" + buffer.get(i).isDone() + "\t" + buffer.get(i).getDescription() + "\t" + temp2.getAt() + "\n");
+                break;
+            }
+        }
+        fw.close();
+    }
+
+    public static void loadFile() throws FileNotFoundException {
+        File f = new File("list.txt"); // create a File for the given file path
+        Scanner s = new Scanner(f); // create a Scanner using the File as the source
+        while (s.hasNext()) {
+            loadTask(s.nextLine());
+        }
+    }
+
+    public static void loadTask(String line){
+        String[] arr = line.split("\t");
```

Comment:
maybe assign arr[0],arr[1],arr[2],arr[3] variable names in order to improve the code's readability?